CHANGELOG.md

```diff
@@ -6,7 +6,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## Unreleased
-
```

Comment:
I think @xrmx is talking about the deleted line L9.

Reply:
yes

Reply:
@keithkroeger could you please fix that?